test/config-webpack-serve.spec.js

```diff
@@ -280,4 +299,29 @@ describe('config webpack serve', () => {
     });
   });
 
+  it('should pass through envid from the command line', () => {
+    argv.envid = 'asdf';
+
+    bindToDone();
+    expect(openCalledWith).toContain(`?envid=asdf`);
+  });
+
+  it('should pass through svcid from the command line', () => {
+    argv.svcid = 'asdf';
+
+    bindToDone();
+    expect(openCalledWith).toContain(`?svcid=asdf`);
+  });
+
+  it('should pass through envid and svcid, but not junk from the command line', () => {
+    argv.envid = 'asdf1';
+    argv.svcid = 'asdf2';
+    argv.myid = 'asdf3';
+
+    bindToDone();
+    expect(openCalledWith).toContain(`envid=asdf1`);
```

Comment:
Since you're testing with multiple query parameters here it might be helpful to write a test that validates whether they all form a valid query string (question mark at the beginning, ampersands between them, etc.)